core/src/avm2/globals.rs

```diff
@@ -70,6 +70,12 @@ pub struct SystemClasses<'gc> {
     pub textformat: ClassObject<'gc>,
     pub graphics: ClassObject<'gc>,
     pub igraphicsdata: ClassObject<'gc>,
+    pub graphicsbitmapfill: ClassObject<'gc>,
```

Comment:
note: we don't really need to add _every_ single class we're ever planning to use in Rust - we can also look them up on demand:
```rs
let filters_namespace = Namespace::package("flash.filters", activation.context.gc_context);
let filter_class = Multiname::new(filters_namespace, "BitmapFilter");
let filter_class_object = activation.resolve_class(&filter_class)?;
```
Not saying which one is correct in this scenario, just a thing to keep in mind when adding them to SystemClasses.

Reply:
I know, but I added these because I know we're going to need them for `drawGraphicsData` and `readGraphicsData`